benches/main.rs

```diff
@@ -201,3 +201,49 @@ fn x_combined(bench: &mut Bencher) {
         black_box(Time::parse_str(&t4).unwrap());
     })
 }
+
+#[bench]
+fn bench_format_date(bench: &mut Bencher) {
```

Comment:
```suggestion
fn format_date(bench: &mut Bencher) {
```

Bit pretty of me, but it's somewhat tautologous to say "bench_" we we know these are benchmarks.